Wikipedia/Code/ArticleCollectionViewController.swift

```diff
@@ -8,7 +8,7 @@ protocol ArticleCollectionViewControllerDelegate: NSObjectProtocol {
 }
 
 @objc(WMFArticleCollectionViewController)
-class ArticleCollectionViewController: ColumnarCollectionViewController, ReadingListHintPresenter, EditableCollection {
+class ArticleCollectionViewController: ColumnarCollectionViewController, ReadingListHintPresenter, EditableCollection, EventLoggingEventValuesProviding, ReadingListsFunnelProvider {
```

Comment:
I'm guessing `provider` and `providing` mean the same thing? If so any thoughts on which one we could standardize on?

Reply:
Oh I guess a `provider` is an instance conforming to `providing`... never mind :)